instructions/day1/Application/README.md

```diff
@@ -50,11 +51,40 @@ _Stepping on the gas pedal triggers a request in the frontend to the API in the
 
 </details>
 
+## Run App in Development Environment
```

Comment:
Love this addition. Could you please mark this as "optional" in the headline and write a small comment that this segment is not required to continue deploying the app?

Thanks.